src/betterproto/__init__.py

```diff
@@ -1532,8 +1532,9 @@ def to_timedelta(self) -> timedelta:
 
     @staticmethod
     def delta_to_json(delta: timedelta) -> str:
-        parts = str(delta.total_seconds()).split(".")
+        parts = f"{delta.total_seconds():f}".split(".")
         if len(parts) > 1:
+            parts[1] = parts[1].rstrip('0')
```

Comment:
```suggestion
            parts[1] = parts[1].rstrip("0")
```
We use black to format code